pkg/ocm/ocmWrapper.go

```diff
@@ -289,6 +290,9 @@ func (*DefaultOCMInterfaceImpl) GetOCMEnvironment() (*cmv1.Environment, error) {
 
 	responseEnv, err := connection.ClustersMgmt().V1().Environment().Get().Send()
 	if err != nil {
```

Comment:
Do we also need to consider an additional check for `responseEnv` being nil to handle cases where an error might prevent `responseEnv` from being populated as expected something like:? 
```
if err != nil {
    var isForbidden bool
    if responseEnv != nil {
        isForbidden = responseEnv.Status() == http.StatusForbidden
        if responseEnv.Error() != nil { 
            isForbidden = isForbidden || responseEnv.Error().Status() == http.StatusForbidden
        }
    }
    
    if isForbidden {
        return nil, fmt.Errorf("user does not have enough permissions to fetch the OCM environment resource. Please ensure you have the necessary permissions or try exporting the BACKPLANE_URL environment variable.\nUnderlying error: %w", err)
    }
    
    return nil, fmt.Errorf("failed to fetch the cluster environment: %w", err)
```

Reply:
`responseEnv` will always be populated as going by the OCM Cluster Service codebase.

But, this is a nice suggestion, better to handle all the edge cases, let me add it.